cpp/src/replace/nans.cu

```diff
@@ -202,7 +204,7 @@ std::unique_ptr<column> normalize_nans_and_zeros(column_view const& input,
 
   // from device. unique_ptr which gets automatically cleaned up when we leave.
   auto out_view = out->mutable_view();
-  normalize_nans_and_zeros(out_view, stream);
+  detail::normalize_nans_and_zeros(out_view, stream);
```

Comment:
I'm a little confused with this part. The enclosing function is in `cudf::detail`. Is `detail::` necessary here? 
The intended target is the preceding function in the same namespace, no?

Reply:
It makes that namespace lookup a bit more explicit. I think this is slightly better than before.